Add About page (#275)
grammyjs/website: site/docs/resources/about.md

# About grammY

## What is grammY?

grammY is a piece of software that you can use when you want to program your own chat bot for [the Telegram messenger](https://telegram.org).
When you make bots, you will notice that some parts of this process are tedious and always the same.
grammY does the heavy lifting for you and makes it super simple to create a bot.

## When was grammY created?

The first publish of grammY code was in late March, 2021.
It reached the first stable version a few weeks later.

## Who develops grammY?

grammY is developed by a team of volunteers who work on the core library, the docs, and the remaining ecosystem in their free time.
You can find a list of all contributors [in the README of our repository](https://github.com/grammyjs/grammY#contributors-).

> Feel free to take contact [in the group chat](https://t.me/grammyjs)!

Author: Make contact? Isn't it called taking contact?

> Feel free to take contact [in the group chat](https://t.me/grammyjs)!

Reviewer: I haven’t heard that.

Notice the difference:

- https://www.merriam-webster.com/dictionary/make%20contact
- https://www.merriam-webster.com/dictionary/take%20contact

Except that, I think making or taking contact is more like physical. Should we use some other term or something?

> Feel free to take contact [in the group chat](https://t.me/grammyjs)!

Author: Huh, TIL. Thanks!

If you speak Russian, you can also join us [here](https://t.me/grammyjs_ru)!

## How is grammY developed?

> grammY is completely free and open-source software.

Author: I'm more familiar with something being "free software", as in the uncountable stuff called OSS, rather than a product. Don't have strong opinions, though, I'll leave it up to you.

Its code is [available on GitHub](https://github.com/grammyjs/grammY).

We welcome all contributions.

> All code changes are reviewed by several pairs of eyes, optimized iteratively and tested extensively, often using production code of various projects and people.

Reviewer: ```suggestion
All code changes are reviewed by several pairs of eyes, optimized iteratively and tested extensively, often using the production code of various projects and people.
```

Author: Same as with software.

## What programming language does grammY use?

grammY is written from the ground up in TypeScript—a superset of JavaScript.
Therefore, it runs on Node.js.

However, grammY can also run on Deno, which positions itself as the successor of Node.js.
(Technically, you can even run grammY on modern browsers, though this will rarely be useful.)

## How does grammY compare to its competitors?

> If you're coming from a different programming language or framework, you can check out our [detailed comparison between frameworks](./comparison.md).

Reviewer: ```suggestion
If you’re coming from a different programming language or framework, you can check out our [detailed comparison between frameworks](./comparison.md).
```

Author: Has this been performed consistently everywhere yet? I think we wanted to have a single PQ that fixes it everywhere at some point

Reviewer: Right, it might not have been related to this very much. But you created a new page, and we want it to be first published with the right quotes. :P

## How can I support you?

There are a number of great ways to support this project!

- Contribute code via a [pull request](https://github.com/grammyjs/grammY/pulls).

> - File [an issue](https://github.com/grammyjs/grammY/issues) about a bug you discovered, a feature you'd like to see, or anything else.

Reviewer: ```suggestion
- [File an issue](https://github.com/grammyjs/grammY/issues/new) about a bug you discovered, a feature you’d like to have, or anything else.
```

Author: So I'll only apply this:
```suggestion
- [File an issue](https://github.com/grammyjs/grammY/issues/new) about a bug you discovered, a feature you'd like to have, or anything else.
```

> - Help out with [the docs](https://github.com/grammyjs/website).

Reviewer: ```suggestion
- Help out with the [Documentation](https://github.com/grammyjs/website).
```

> - Assist people in the community chats for [English](https://t.me/grammyjs) or [Russian](https://t.me/grammyjs_ru).

Reviewer: ```suggestion
- Assist people in the community chats in [English](https://t.me/grammyjs) or [Russian](https://t.me/grammyjs_ru).
```

- Or just come tell us what you think!

>   Already leaving a few words about your favorite feature help us determine the direction of the project.

Reviewer: ```suggestion
  Leaving a few words about your favorite feature, helps us determine the direction of the project.
```
